packages/elements/src/components/ino-img/ino-img.scss

```diff
@@ -30,4 +29,34 @@ ino-img {
   &.mdc-image-list__item {
     display: list-item;
   }
+
+  img:before {
```

Comment:
Hi @P1NHE4D is there a reason why we need the pseudo before element for the image?

Reply:
The pseudo element is only displayed if the image fails to load. As far as I remember, I only added it to hide the default icon and alt text in case of an error while loading the image.